### PR TITLE
snag-1.2.0

### PR DESCRIPTION
**Pull request by zabawaba99**
### All Submissions:

- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew/pulls) for the same update/change?

### CODE REVIEW

File: Library/Formula/snag.rb [hunk 1]
```diff
@@ -16,46 +16,19 @@ class Snag < Formula
 
   depends_on "go" => :build
 
-  go_resource "gopkg.in/yaml.v2" do
```
Comment (DomT4) [inline Library/Formula/snag.rb]: I presume all these are vendored now if we're deleting them from the formulae? There's no external resource fetching during compile happening?

File: Library/Formula/snag.rb [hunk 2]
```diff
@@ -16,46 +16,19 @@ class Snag < Formula
...
-  end
-
   def install
     ENV["GOPATH"] = buildpath
 
-    snagpath = buildpath/"src/github.com/Tonkpils/snag"
-    snagpath.install Dir["{*,.git}"]
+    mkdir_p buildpath/"src/github.com/Tonkpils/"
+    ln_s buildpath, buildpath/"src/github.com/Tonkpils/snag"
     Language::Go.stage_deps resources, buildpath/"src"
```
Comment (DomT4) [inline Library/Formula/snag.rb]: Also, if we can remove all the resources, you can kill this line as well now.

File: Library/Formula/snag.rb [hunk 3]
```diff
@@ -16,46 +16,19 @@ class Snag < Formula
...
-    url "https://github.com/fatih/color.git",
-      :revision => "2f376994f3b3cae1602f63eb5bdfb58101a94a08"
-  end
-
   def install
     ENV["GOPATH"] = buildpath
 
-    snagpath = buildpath/"src/github.com/Tonkpils/snag"
-    snagpath.install Dir["{*,.git}"]
+    mkdir_p buildpath/"src/github.com/Tonkpils/"
```
Comment (bfontaine) [inline Library/Formula/snag.rb]: Use `(buildpath/"src/github.com/Tonkpils/").mkpath` instead of `mkdir_p`.

File: Library/Formula/snag.rb (hunk 1 above)
Comment (zabawaba99) [inline Library/Formula/snag.rb]: Yes, this new version of `snag` has all it's dependencies vendored. Nothing is fetched during compilation.

File: Library/Formula/snag.rb (hunk 2 above)
Comment (zabawaba99) [inline Library/Formula/snag.rb]: Removed. :+1:

File: Library/Formula/snag.rb (hunk 3 above)
Comment (zabawaba99) [inline Library/Formula/snag.rb]: Just changed it. What's the difference between the two? Asking for future reference.

Comment (DomT4) [inline Library/Formula/snag.rb]: Just looks a bit neater :smile:

Comment (zabawaba99) [inline Library/Formula/snag.rb]: Haha, can't argue with that logic!

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/b823158e0950c841373f5567c2e13aec0c5296f8. Thank you for another contribution to Homebrew @zabawaba99! :smiley_cat:

**Pull request closed without merging**